packages/cli/test/integration/owner.endpoints.test.ts

```diff
@@ -25,7 +25,13 @@ beforeAll(async () => {
 });
 
 beforeEach(async () => {
+	jest.isolateModules(() => {
+		jest.mock('../../config');
+	});
```

Comment:
From my tests, `isolateModules` is unneeded and was therefore removed at #3066. The mock is also unneeded it seems.